xdsl/dialects/x86/ops.py

```diff
@@ -815,224 +848,223 @@ def parse(cls, parser: Parser) -> Self:
         return op
 
 
-class RRROperation(
+class RSS_Operation(
     Generic[R1InvT, R2InvT, R3InvT], X86Instruction, X86CustomFormatOperation, ABC
 ):
     """
     A base class for x86 operations that have three registers.
     """
 
-    r1 = operand_def(R1InvT)
-    r2 = operand_def(R2InvT)
-    r3 = operand_def(R3InvT)
-
-    result = result_def(R1InvT)
+    register_in = operand_def(R1InvT)
+    register_out = result_def(R1InvT)
+    source1 = operand_def(R2InvT)
+    source2 = operand_def(R3InvT)
 
     def __init__(
         self,
-        r1: Operation | SSAValue,
-        r2: Operation | SSAValue,
-        r3: Operation | SSAValue,
+        register_in: Operation | SSAValue,
+        source1: Operation | SSAValue,
+        source2: Operation | SSAValue,
         *,
         comment: str | StringAttr | None = None,
-        result: R1InvT,
+        register_out: R1InvT,
     ):
         if isinstance(comment, str):
             comment = StringAttr(comment)
 
         super().__init__(
-            operands=[r1, r2, r3],
+            operands=[register_in, source1, source2],
             attributes={
                 "comment": comment,
             },
-            result_types=[result],
+            result_types=[register_out],
         )
 
     def assembly_line_args(self) -> tuple[AssemblyInstructionArg | None, ...]:
-        return self.r1, self.r2, self.r3
+        return self.register_in, self.source1, self.source2
 
 
 # endregion
 
 
 @irdl_op_definition
-class RR_AddOp(R_RR_Operation[GeneralRegisterType, GeneralRegisterType]):
+class RS_AddOp(RS_Operation[GeneralRegisterType, GeneralRegisterType]):
     """
-    Adds the registers r1 and r2 and stores the result in r1.
+    Adds the registers r and s and stores the result in r.
```

Comment:
You're going to hate me but this would make more sense if the operands/results were called `r` and `s`

Reply:
Train has left the station